tests/Unit/Services/VoucherEvaluator/VoucherEvaluatorTest.php

```diff
@@ -170,74 +264,34 @@ public function itCreditsWhenAChildIsUnderSchoolAge()
-        $this->assertContains(self::CREDIT_TYPES['ChildIsUnderSchoolAge'], $credits, '');
+        $this->assertContains(self::CREDIT_TYPES['ChildIsBetweenOneAndPrimarySchoolAge'], $credits, '');
         $this->assertEquals(3, $evaluation->getEntitlement());
     }
 
     /** @test */
-    public function itCreditsWhenAChildIsUnderSecondarySchoolAge()
+    public function itDoesNotCreditWhenAChildisSecondarySchoolAge()
     {
-        // Make a Child under School Age.
-        $child = factory(Child::class, 'underSchoolAge')->make();
+        // Make a Secondary school child
+        $child = factory(Child::class, 'isSecondarySchoolAge')->make();
 
-        // Make standard evaluator
-        $evaluator = EvaluatorFactory::make('extended_age');
-        $evaluation = $evaluator->evaluate($child);
-        $credits = $evaluation["credits"];
+        $rulesMod = collect($this->rulesMods["credit-extended"]);
 
-        // Check there's one, because child is not under one.
-        $this->assertEquals(1, count($credits));
-
-        // Check the correct credit type is applied.
-        $this->assertNotContains(self::CREDIT_TYPES['ChildIsUnderOne'], $credits, '');
-        $this->assertContains(self::CREDIT_TYPES['ChildIsUnderSecondarySchoolAge'], $credits, '');
-        $this->assertEquals(3, $evaluation->getEntitlement());
-    }
-
-    /** @test */
-    public function itDoesNotCreditWhenAChildIsOverSchoolAge()
-    {
-        // Make a Child under School Age.
-        $child = factory(Child::class, 'overSchoolAge')->make();
-
-        // Make standard evaluator
-        $evaluator = EvaluatorFactory::make();
-        $evaluation = $evaluator->evaluate($child);
-        $credits = $evaluation["credits"];
-
-        // Check there's one, because child is not under one.
-        $this->assertEquals(0, count($credits));
-
-        // Check the correct credit type is applied.
-        $this->assertNotContains(self::CREDIT_TYPES['ChildIsUnderOne'], $credits);
-        $this->assertNotContains(self::CREDIT_TYPES['ChildIsUnderSchoolAge'], $credits);
-        $this->assertEquals(0, $evaluation->getEntitlement());
-    }
-
-    /** @test */
-    public function itDoesNotCreditWhenAChildIsOverSecondarySchoolAge()
-    {
-        // Make a Child under School Age.
-        $child = factory(Child::class, 'overSecondarySchoolAge')->make();
-
-        // Make standard evaluator
-        $evaluator = EvaluatorFactory::make('extended_age');
+        // Make extended evaluator
+        $evaluator = EvaluatorFactory::make($rulesMod);
         $evaluation = $evaluator->evaluate($child);
         $credits = $evaluation["credits"];
 
-        // Check there's one, because child is not under one.
+        // Check there's none, because child is not primary or under.
         $this->assertEquals(0, count($credits));
 
         // Check the correct credit type is applied.
         $this->assertNotContains(self::CREDIT_TYPES['ChildIsUnderOne'], $credits);
```

Comment:
Since there are no credits, are these NotContains relevant to test here?

Reply:
aha! true! will remove